csharp_dotnetcore/14.NLP-With-Dispatch/NLP-With-Dispatch-Bot.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <AssemblyName>NLP_With_Dispatch_Bot</AssemblyName>
+    <RootNamespace>NLP_With_Dispatch_Bot</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <CodeAnalysisRuleSet>NLP-With-Dispatch-Bot.ruleset</CodeAnalysisRuleSet>
+  </PropertyGroup>
```

Comment:
Move this CodeAnalysisRuleSet into the main Property Group. No need to have the Release/Debug conditionals.